README.md

```diff
@@ -0,0 +1,10 @@
+# movies-series SQL analysis
+Analyzing movies and series dataset to answer real life questions.
+# Description
+The purpose of this SQL project was to analyze movies and series from netflix data.
+To achieve the project's goals, a series of SQL queries were written to extract and analyze relevant data. These queries included aggregations to identify the most popular movies or series, as well as joins.
+# How to install
+1. Install to your computer PostgreSQL.
+2. Go to Kaggle website and search for: Latest Netflix data with 26+ joined attributes, downloade the csv to your computer.
```

Comment:
Why do we need to download from kaggle, when we have it in the datasets/CSV files in the repository?